version_policy.rb

```diff
@@ -89,6 +89,10 @@
   "unicode-display_width",
   "varia_model",
   "httpclient",
+  "chef", # should be maintained by update_current_chef task but that doesn't
```

Comment:
Just had a chat with @jkeiser - this actually seems like the correct solution.  I'll update the comments and Julia wiki page with more information.